src/static/js/routes/the-bureau/index.js

```diff
@@ -16,8 +16,8 @@ function init() {
     return;
   }
 
-  // MobileCarousel with 599px breakpoint.
-  var mobileCarousel = new MobileCarousel( 599 );
+  var breakpointPx = 599;
```

Comment:
As mentioned in convo, sounds like a good idea, but out of scope with this PR. Opened https://github.com/cfpb/cfgov-refresh/issues/726